README.md

```diff
@@ -1,18 +1,25 @@
 # Trial task for back-end developers
 
-Please use Node.js 12+, MySQL and a web framework of your choice to implement the following trial task. You can use TypeScript and any node libraries you think are necessary or helpful for you, but it helps us a lot if you keep it minimal. It is recommended to use some code-formatter and linter of your choice (we recommend [Prettier](https://prettier.io/) and [XO](https://github.com/xojs/xo)).
+Someone from support notified us that users are using too many times passwords like "password", "dog123", or "heeello" and their accounts are getting hacked by their neighbours to see private pictures of their private parts.
+We need this fixed ASAP, but our developers are busy working on the next-cool-feature™ using the next-cool-stack™, and they have no time for implementing the changes needed to fix the issue. So our only chance is YOU!*
+
+Your objective for this task is to build two components:
+a REST API that receives a password and gives information about its validity, given a set of rules
+a script that reads the already existing passwords from the database and sets a validity flag based on the API you implement
+
+We ask you to use Node.js 12+ (TypeScript is an option), MySQL and a web framework of your choice to implement the following trial task. You can use TypeScript and any node libraries you think are necessary or helpful for you, but it helps us a lot if you keep it minimal. We recommend using some code-formatter and linter of your choice (we recommend [Prettier](https://prettier.io/) and [XO](https://github.com/xojs/xo)).
 
 What we value when reviewing your task:
 - attention to perfomance and scalability
 - a good README with instructions on how to run your execution
 - good code architecture
 - polished result: not too many typos or commented code and consistent styling
```

Comment:
Nitpick to avoid ambiguity: consider "consistent styling, and not too many typos or commented code"

Reply:
fixed